R/resample.R

```diff
@@ -95,19 +94,48 @@ fit_resamples.formula <- function(formula,
                                   metrics = NULL,
                                   control = control_resamples()) {
 
+  lifecycle::deprecate_soft("0.0.2",
```

Comment:
Can you make the deprecation deadline 0.1.2 since I'll be upping the version number to 0.1.0?

Reply:
This is not a deadline but the date at which the function is deprecated. This should be the same version for the whole lifecycle of deprecation (soft-deprecated, deprecated, defunct).

Reply:
For clarification here @topepo the next version on CRAN is going to be 0.1.0? Are we going from 0.0.1 straight to 0.1.0? I believe then this should be 0.1.0, because that is when the function is soft deprecated.

Reply:
I set this as 0.1.0 in 36bb83455820914d101b33f65c25bd8a12878e00. Let me know if I misinterpreted.